packages/application-shell-connectors/src/components/application-state/README.md

```diff
@@ -0,0 +1,31 @@
+# Connectors for `applicationState`
+
+This component exposes getters/setters to set and retrieve the `applicationState`.
+
+> It uses the new React Context API, so you need to provide the `ApplicationStateProvider` up in the tree. This is done within the AppShell!
+
+Additionally it provides a HOC to inject the `applicationState`.
+
+Use this component to access about the `user`, `project` and application `environment`.
```

Comment:
*Update*: After reading the code I now get that it's "just" `window.app`. A comment here explaining that it holds that value - and why one would use the consumer instead of directly accessing `window.app` might be useful. I'd find it useful if the comment could also explain which values `window.app` contains and what they're used for.

Reply:
Yeah, I can list some of the values, however, `window.app` can contain application specific values which does not make sense to document.

Reply:
Sure :) My intention was to document that it contains `window.app` and why one would use `environment` over just accessing `window.app` directly - and to mention that the values can be custom as well as mentioning the known ones.